### DISCUSSION TITLE
What do these two messages mean? mangle parameter and headerIds

### DISCUSSION THREAD

**Discussion by busomyeong** [markedjs/marked#2825] [Q&A]
marked(): mangle parameter is enabled by default, but is deprecated since version 5.0.0, and will be removed in the future. To clear this warning, install https://www.npmjs.com/package/marked-mangle, or disable by setting `{mangle: false}`.

marked(): headerIds and headerPrefix parameters enabled by default, but are deprecated since version 5.0.0, and will be removed in the future. To clear this warning, install  https://www.npmjs.com/package/marked-gfm-heading-id, or disable by setting `{headerIds: false}`.

The content of my markdown file is just "# hello world" for a test. What is the mangle parameter in marked and what is headerIds?

**Comment by UziTech** [comment]
see https://marked.js.org/using_advanced#options
